adoc-mode.el

```diff
@@ -133,7 +126,8 @@ configuration file."
           (string :tag "level 1")
           (string :tag "level 2")
           (string :tag "level 3")
-          (string :tag "level 4") )
+          (string :tag "level 4")
+          (string :tag "level 5"))
```

Comment:
Added level 5 from [AsciiDoc documentation](https://docs.asciidoctor.org/asciidoc/latest/sections/titles-and-levels/).